### PR TITLE
Added issues from my peer testing

### PR DESCRIPTION
**Pull request by Nick-McGee**

### CODE REVIEW

File: peerTestingReport/issuesNickMcGee.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Severity: 2 (Causes slight confusion, doesn’t break the system)
+Violation: Aesthetic design and Visibility
+Description: When the user first enters the code editor, there is a built in demo area. This should be removed, as users often thought this was where they are meant to code.
+Modification: Remove this read only area, as it is not needed.
+
+Issue: Users felt the need for a reset button.
+Severity: 2 (Can break the lesson if the world is not reset. This would be a 3, but may not be needed in final product)
+Violation: Help users recover from errors.
+Description: When the Agent mines away at the mining area, it does not regenerate. If the user incorrectly mines the area, they are unable to reset it back to how it was.
+Modification: Using an in-game command block, we would be able to reset the mining area.
```
Comment (Ansper11) [inline peerTestingReport/issuesNickMcGee.txt:11]: this is a good idea

File: peerTestingReport/issuesNickMcGee.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Issue: User gets confused with the ‘read only’ code area.
+Severity: 2 (Causes slight confusion, doesn’t break the system)
```
Comment (Ansper11) [inline peerTestingReport/issuesNickMcGee.txt:2]: I agree

**Comment by yyht6300a** [comment]
Thanks for your work.

File: peerTestingReport/issuesNickMcGee.txt (hunk 1 above)
Comment (eloiseespel) [reply peerTestingReport/issuesNickMcGee.txt:11]: That could be a great idea of doing it as resetting directly in the code builder would require access to the source code

File: peerTestingReport/issuesNickMcGee.txt (hunk 2 above)
Comment (eloiseespel) [reply peerTestingReport/issuesNickMcGee.txt:2]: Yes my users got confused with that

**Pull request merged**